packages/core/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "lib": [ "dom", "es2022" ],               /* Specify library files to be included in the compilation. */
+    "allowJs": true,
+    "outDir": "dist",
```

Comment:
I was having some issues with allowJs and for unclear reasons specifying this solved them. See https://stackoverflow.com/questions/42609768/typescript-error-cannot-write-file-because-it-would-overwrite-input-file